index.js

```diff
@@ -56,12 +56,10 @@ const registerPluginThen = (currentPromise, config) => {
  * @return {null} none
  */
 const handleAllPluginsRegistered = () => {
-    const authUtils = require('./lib/utils/authentication.js')(server);
-
     http.auth.strategy(
         'default',
         'hawk',
-        { getCredentialsFunc: authUtils.getHawkCredentials }
+        { getCredentialsFunc: server.plugins.utilities.auth.getHawkCredentials }
```

Comment:
I also modified auth utilities to be a plugin so that we can access it without requiring it.